app/assets/scss/_search-page.scss

```diff
@@ -95,7 +92,7 @@
   .dm-filters {
```

Comment:
I think we also have filters in briefs front-end now? So would be good to get these styles into the toolkit to ensure consistency. Not sure if the front-end mission is doing this?